locales/de/de.json

```diff
@@ -613,19 +613,19 @@
     "The provided password was incorrect.": "Das angegebene Passwort war falsch.",
     "The provided two factor authentication code was invalid.": "Der angegebene Zwei-Faktor-Authentifizierungscode war ungültig.",
     "The provided VAT number is invalid.": "Die angegebene Umsatzsteuer-Identifikationsnummer ist ungültig.",
-    "The receipt emails must be valid email addresses.": "Die Belege E-Mails müssen gültige E-Mail-Adressen sein.",
+    "The receipt emails must be valid email addresses.": "Die Beleg-E-Mail-Adressen müssen gültige E-Mail-Adressen sein.",
     "The resource was attached!": "Die Ressource wurde angehängt!",
     "The resource was prevented from being saved!": "Die Ressource konnte nicht gespeichert werden!",
     "The resource was updated!": "Die Ressource wurde aktualisiert!",
     "The selected country is invalid.": "Das gewählte Land ist ungültig.",
     "The selected plan is invalid.": "Der gewählte Plan ist ungültig.",
-    "The team's name and owner information.": "Der Name des Teams und Informationen zum Besitzer.",
+    "The team's name and owner information.": "Teamname und Informationen zum Besitzer",
```

Comment:
```suggestion
    "The team's name and owner information.": "Teamname und Informationen zum Besitzer.",
```

Reply:
Periods in this context don't make much sense (since they are no full sentence), but it's probably better to keep the original format!

Reply:
Quite possibly. This key is used in [Laravel Jetstream](https://github.com/laravel/jetstream/blob/2.x/stubs/inertia/resources/js/Pages/Teams/Partials/UpdateTeamNameForm.vue#L33-L35).

But it's better to stick to the existing message format.